### PR TITLE
Salesforce 2.3

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: salesforce/latest/using/community.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+## Lightning Experience
+
+1. See [Create an Experience Cloud Site pg. 51](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+2. See [Add Members to Your Experience Cloud Site pg. 54](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+3. [Share content externally to community members Lightning Experience](#share-content-externally-to-community-members-lightning-experience)
+
+## Share content externally to community members Lightning Experience
```
Comment (pnirwal) [inline salesforce/latest/using/community.md:17]: Please capitalise "Community members".

File: salesforce/latest/using/community.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+1. Log in to Salesforce.
+
+2. Log in as a community administrator to Alfresco Share in the Canvas app.
```
Comment (pnirwal) [inline salesforce/latest/using/community.md:21]: Remove word "community". 
"Log in as an administrator to Alfresco Share in the Canvas app."

File: salesforce/latest/using/community.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+2. See [Add Members to Your Experience Cloud Site pg. 54](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+3. [Share content externally to community members Salesforce Classic](#share-content-externally-to-community-members-salesforce-classic)
+
+## Share content externally to community members Salesforce Classic
+
+1. Log in to Salesforce.
+
+2. Click **Alfresco Repository** and log in as a community administrator to Alfresco Share in the Canvas app.
```
Comment (pnirwal) [inline salesforce/latest/using/community.md:79]: Remove word "community".
.... log in as an administrator to Alfresco Share in the Canvas app.

File: salesforce/latest/using/community.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+      ![classic-share-externally]({% link salesforce/images/classic-share-externally.png %})
+
+11. Select the **Salesforce Community Name** you would like to share with and the **Users**. Click **Save**.
+
+      ![lightning-externally-select]({% link salesforce/images/lightning-externally-select.png %})
+
+You will see a **Successfully Shared Accounts Externally** message. The result of the performed actions is that when the community member logs into Salesforce they will see the case in their **All Open Cases** list.
+
+## Retrieving shared cases
```
Comment (pnirwal) [inline salesforce/latest/using/community.md:121]: Update section name to "Salesforce Community member access"

File: salesforce/latest/using/community.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+2. Click the **My Open Cases** drop down list and then select **All Open Cases**.
+
+      ![lightning-open-cases]({% link salesforce/images/lightning-open-cases.png %})
+
+3. Click the link of the case number that has been shared with you to open it.
+
+      ![lightning-all-new]({% link salesforce/images/lightning-all-cases.png %})
+
+4. Log in as a Community member to access the case.
```
Comment (pnirwal) [inline salesforce/latest/using/community.md:133]: Update to: 
"Log in as a Community member to access the case and documents linked to the case.

File: salesforce/latest/using/community.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+      ![lightning-alfresco-tab]({% link salesforce/images/lightning-alfresco-tab.png %})
```
Comment (pnirwal) [inline salesforce/latest/using/community.md:135]: At the end of this section add: 

Community members can download and view documents linked to the case. They can also upload documents to the case that will get stored in Alfresco.

**Review by pnirwal — CHANGES_REQUESTED** [review]
Please review my 17 comments! Thanks.

File: salesforce/latest/config/index.md [hunk 7]
```diff
@@ -123,3 +123,27 @@ Ensure you have the [prerequisites](#prereqs) installed and configured first, an
     By doing this when a user is created your domain name is used by default instead of `@myDomain.com`.
 
     >**Note:** The Apex code can be configured in lots of different ways to suit your organization. See the Apex documentation at Salesforce for more: [What is Apex?](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_intro_what_is_apex.htm){:target="_blank"}.
+
+## Configure a Salesforce Community site
+
+To configure a Salesforce Community site you must add the administrator user of Alfresco Content Services that you use as an administrator of Salesforce to the `GROUP_SALESFORCE_MODERATORS` group in Alfresco Content Services.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply salesforce/latest/config/index.md:129]: done

File: salesforce/latest/config/index.md [hunk 8]
```diff
@@ -123,3 +123,27 @@ Ensure you have the [prerequisites](#prereqs) installed and configured first, an
...
+
+> **Note:** Support for Salesforce Community is only available when using Alfresco Cloud (Digital Business Platform-as-a-Service (PaaS)).
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply salesforce/latest/config/index.md:131]: done

File: salesforce/latest/install/index.md [hunk 9]
```diff
@@ -17,9 +17,11 @@ There are a number of software requirements for using the Salesforce Connector.
 
 ### Alfresco requirements
 
-* Alfresco Content Services 6.2.1 or later
+* Alfresco Content Services 6.2.2 or later
 * Identity Service 1.3 or later (if you plan to use Single Sign On (SSO))
 
+> **Note:** Support for Salesforce Community is only available when using Alfresco Cloud (Digital Business Platform-as-a-Service (PaaS)).
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply salesforce/latest/install/index.md:23]: done

File: salesforce/latest/support/index.md [hunk 10]
```diff
@@ -8,5 +8,6 @@ The following are the supported platforms for the Content Connector for Salesfor
 | ------- | ----- |
 | Content Services 7.0.x | |
 | Content Services 6.2.2 | |
-| Content Services 6.2.1 | |
 | Identity Service 1.3 or above | If using Single Sign On (SSO) |
+
+> **Note:** Support for Salesforce Community is only available when using Alfresco Cloud (Digital Business Platform-as-a-Service (PaaS)).
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply salesforce/latest/support/index.md:13]: done

File: salesforce/latest/using/community.md (hunk 2 above)
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md:21]: done

File: salesforce/latest/using/community.md (hunk 3 above)
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md:79]: done

File: salesforce/latest/using/community.md (hunk 4 above)
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md:121]: done

File: salesforce/latest/using/community.md (hunk 5 above)
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md:133]: done

File: salesforce/latest/using/community.md (hunk 6 above)
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md:135]: done

File: salesforce/latest/using/community.md [hunk 11]
```diff
@@ -0,0 +1,135 @@
+---
+title: Community Sites
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md:2]: done

File: salesforce/latest/using/community.md (hunk 1 above)
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md:17]: done

File: _data/toc/salesforce.yaml [hunk 12]
```diff
@@ -13,8 +13,26 @@
           path: '/salesforce/latest/using/'
         - title: 'Recommended content'
           path: '/salesforce/latest/using/recommend/'
+        - title: 'Community Sites'
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply _data/toc/salesforce.yaml:16]: done

File: salesforce/latest/using/community.md [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+
+## Salesforce Classic
+
+1. See [Create an Experience Cloud Site pg. 51](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+2. See [Add Members to Your Experience Cloud Site pg. 54](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+3. [Share content externally to community members Salesforce Classic](#share-content-externally-to-community-members-salesforce-classic)
+
+## Share content externally to community members Salesforce Classic
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md:75]: done

File: salesforce/latest/using/community.md [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+
+> **Note:** Support for Salesforce Community is only available when using Alfresco Cloud (Digital Business Platform-as-a-Service (PaaS)).
+
+## Lightning Experience
+
+1. See [Create an Experience Cloud Site pg. 51](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+2. See [Add Members to Your Experience Cloud Site pg. 54](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+3. [Share content externally to community members Lightning Experience](#share-content-externally-to-community-members-lightning-experience)
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md:15]: done

File: salesforce/latest/using/community.md [hunk 15]
```diff
@@ -0,0 +1,135 @@
+---
+title: Community Sites
+---
+
+To share content externally to a community site and retrieve a shared case as a community member you need to peform a series of actions. Sharing sites externally can be done using either the [Lightning Experience](#lightning-experience) or [Salesforce Classic](#salesforce-classic). [Retrieving shared cases](#retrieving-shared-cases) as a community member can only be done using the Lightning Experience. Before you begin ensure you have added the administrator user of Alfresco Content Services that you use as an administrator of Salesforce to the `GROUP_SALESFORCE_MODERATORS` group in Alfresco Content Services, for more see [Configure a Salesforce Community site]({% link salesforce/latest/config/index.md %}#configure-a-salesforce-community-site).
+
+> **Note:** Support for Salesforce Community is only available when using Alfresco Cloud (Digital Business Platform-as-a-Service (PaaS)).
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md:7]: done

File: salesforce/latest/using/community.md [hunk 16]
```diff
@@ -0,0 +1,135 @@
+---
+title: Community Sites
+---
+
+To share content externally to a community site and retrieve a shared case as a community member you need to peform a series of actions. Sharing sites externally can be done using either the [Lightning Experience](#lightning-experience) or [Salesforce Classic](#salesforce-classic). [Retrieving shared cases](#retrieving-shared-cases) as a community member can only be done using the Lightning Experience. Before you begin ensure you have added the administrator user of Alfresco Content Services that you use as an administrator of Salesforce to the `GROUP_SALESFORCE_MODERATORS` group in Alfresco Content Services, for more see [Configure a Salesforce Community site]({% link salesforce/latest/config/index.md %}#configure-a-salesforce-community-site).
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md:5]: Done

**Review by pnirwal — APPROVED** [review]
Looks good!

File: salesforce/latest/using/community.md [hunk 17]
```diff
@@ -0,0 +1,139 @@
+---
+title: Salesforce Community
+---
+
+Support for Salesforce Community allows external Salesforce Community Members to store and access documents in Alfresco. To provide access and share documents with external Salesforce Community Members, internal Salesforce users need to perform a series of actions.
+Before you begin, ensure your internal Salesforce users have been added to the `GROUP_SALESFORCE_MODERATORS` group in Alfresco Content Services, for more see [Configure Salesforce Community]({% link salesforce/latest/config/index.md %}#configure-a-salesforce-community-site).
```
Comment (gravitonian) [inline salesforce/latest/using/community.md:6]: [Configure Salesforce Community]({% link salesforce/latest/config/index.md %}#configure-a-salesforce-community-site should be [Configure Salesforce Community]({% link salesforce/latest/config/index.md %}#configure-a-salesforce-community

**Review by gravitonian — CHANGES_REQUESTED** [review]
Went through the Community related stuff and tested all links. Just one link needs updating.

**Review by anxumalo — COMMENTED** [review]
We should also add the new release to the ACS 6.2.2 SP page

**Comment by oliveralfresco2018** [comment]
Added to the 6.2.2 SP page also

**Review by gravitonian — DISMISSED** [review]
Built and checked link, looks ok.

File: salesforce/latest/using/community.md [hunk 18]
```diff
@@ -0,0 +1,139 @@
...
+
+      ![classic-share-externally]({% link salesforce/images/classic-share-externally.png %})
+
+11. Select the **Salesforce Community Name** you would like to share with and the **Users**. Click **Save**.
+
+      ![lightning-externally-select]({% link salesforce/images/lightning-externally-select.png %})
+
+You will see a **Salesforce Community Member Access** message. The result of the performed actions is that when the Community Member logs into Salesforce they will see the case in their **All Open Cases** list.
+
+## Retrieving shared cases
```
Comment (pnirwal) [inline salesforce/latest/using/community.md:123]: Minor update, maybe change to "Community Member accessing shared content"

**Review by pnirwal — CHANGES_REQUESTED** [review]
One minor update!

File: salesforce/latest/using/community.md (hunk 18 above)
Comment (oliveralfresco2018) [reply salesforce/latest/using/community.md]: Changed cheers

**Review by pnirwal — APPROVED** [review]
Looks goood!

**Review by pnirwal — APPROVED** [review]
Looks good!

**Review by gravitonian — APPROVED** [review]

**Pull request merged**